tests/formats/test_ascii.py

```diff
@@ -108,28 +109,30 @@ async def test_export_multiple_types(rc: RawConfig):
         '\n'
         '\n'
         'Table: example/A\n'
-        '---------  ---  ---------  -----\n'
-        '_type      _id  _revision  value\n'
-        'example/A  ∅    ∅          1\n'
-        'example/A  ∅    ∅          2\n'
-        'example/A  ∅    ∅          3\n'
-        '---------  ---  ---------  -----\n'
+        '---------  ---  ---------  -----  \n'
+        '_type      _id  _revision  value  \n'
+        'example/A  ∅    ∅          1    \n'
+        'example/A  ∅    ∅          2    \n'
+        'example/A  ∅    ∅          3    \n'
+        '---------  ---  ---------  -----  \n'
+        '\n'
         '\n'
         'Table: example/B\n'
-        '---------  ---  ---------  -----\n'
-        '_type      _id  _revision  value\n'
-        'example/B  ∅    ∅          1\n'
-        'example/B  ∅    ∅          2\n'
-        'example/B  ∅    ∅          3\n'
-        '---------  ---  ---------  -----\n'
+        '---------  ---  ---------  -----  \n'
+        '_type      _id  _revision  value  \n'
+        'example/B  ∅    ∅          1    \n'
+        'example/B  ∅    ∅          2    \n'
+        'example/B  ∅    ∅          3    \n'
+        '---------  ---  ---------  -----  \n'
+        '\n'
         '\n'
         'Table: example/C\n'
-        '---------  ---  ---------  -----\n'
-        '_type      _id  _revision  value\n'
-        'example/C  ∅    ∅          1\n'
-        'example/C  ∅    ∅          2\n'
-        'example/C  ∅    ∅          3\n'
-        '---------  ---  ---------  -----'
+        '---------  ---  ---------  -----  \n'
+        '_type      _id  _revision  value  \n'
+        'example/C  ∅    ∅          1    \n'
+        'example/C  ∅    ∅          2    \n'
+        'example/C  ∅    ∅          3    \n'
+        '---------  ---  ---------  -----  \n'
```

Comment:
Maybe it is possible to avoid spaces at the end of line? This will save some bytes transferred data.